### ISSUE TITLE
Angular "Switch to ESBuild" - wrong webdir

### ISSUE THREAD

**Issue by sake12** [ionic-team/vscode-extension#149]
Hi!
After using the "Switch to ESBuiild" Recommendations the path in `capacitor.config.ts` is set to `www\browser`
```
import { CapacitorConfig } from '@capacitor/cli';

const config: CapacitorConfig = {
  appId: 'io.ionic.starter',
  appName: 'mobile',
  webDir: 'www\browser',
  server: {
    androidScheme: 'https'
  }
};

export default config;
```

Which gives error in console when I run `npx cap sync android`:
```
[error] Could not find the web assets directory: .\wwrowser.
        Please create it and make sure it has an index.html file. You can change the path of this directory in
        capacitor.config.ts (webDir option). You may need to compile the web assets for your app (typically npm run
        build). More info: https://capacitorjs.com/docs/basics/workflow#sync-your-project
```

Should be: `webDir: 'www\\browser',`

**Comment by dtarnawsky** [comment]
The extension performs this operation on the webDir:
```typescript
            if (!value.endsWith(sep)) {
              value = join(value, 'browser'); // Angular now writes to a browser folder
            }
```

It is not possible for it to change to `.\wwrowser` and in testing this correctly adds 'browser' as a subfolder of the `webDir` property.

I'll close this issue for now as it appears to be working as expected.

**Comment by sake12** [comment]
It adds correctly, but I think `\b` it treated as a special character: backspace.
That's why the 3rd `w` is removed in `.\wwrowser`.

Maybe it is just me. I'm using Windows Terminal and new PowerShell.

Even VSCode highlights it as a special char:
![image](https://github.com/ionic-team/vscode-extension/assets/6509232/42202641-27f1-4c33-96a3-4239e5cd8dd1)

**Comment by dtarnawsky** [comment]
Ahh that makes sense. Consider this issue fixed in v1.65.1 being released now.
